README.md

```diff
@@ -23,15 +26,53 @@ Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master bra
 
 Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
 
-## Usage
+### Usage for combined policy statements
 
-### Put an example usage of the module here
+To include a policy in your combined policy block, set it to `true`. Otherwise omit the policy variable.
 
 ```hcl
-module "example" {
-  source = "terraform/registry/path"
+// TODO: make sure these references to module names are correct once pushed to the registry
+module "ou_scp" {
+  source = "trussworks/ou-scp"
```

Comment:
```suggestion
  source = "trussworks/ou-scp/aws"
```

My experience is that the naming convention of the repo maps to the source module name.

Reply:
thought it would be ehere.... https://github.com/trussworks/Engineering-Playbook/tree/master/infrasec/tf#going-public but I think Chris is right.

Reply:
Chris is definitely correct. You can look at any of the other module calls in our Terraform codebase.

Reply:
I'm getting the vague sense that Chris might be right about this...